app/homepage.py

```diff
@@ -23,16 +23,42 @@
 # Importing plotly fig objects from graphs.py
 from graphs import generate_meteo_fig
 
+# Importing utils fetched API data
+from utils import build_live_alerts_metadata
+
 
 # ------------------------------------------------------------------------------
 # Before moving to the app layout
 
+#Fetching reusable alert metadata
+alert_metadata = build_live_alerts_metadata()
+if alert_metadata:
+    frame_url = alert_metadata["media_url"]
```

Comment:
FRAME_URL: quand on définit une variable globale, on met en maj ?

Reply:
Should not have let this `if alert_metadata:`, thanks for pointing this out